mobile/src/main/java/org/openhab/habdroid/ui/OpenHABMainActivity.java

```diff
@@ -1401,6 +1392,16 @@ private void loadDrawerItems() {
             mDrawerItemList.add(OpenHABDrawerItem.menuItem("Discovery", getResources().getDrawable(R.drawable.ic_track_changes_grey600_36dp), DRAWER_INBOX));
             mDrawerItemList.add(OpenHABDrawerItem.menuItem("Bindings", getResources().getDrawable(R.drawable.ic_extension_grey600_36dp), DRAWER_BINDINGS));
         }
+        Drawable aboutDrawable = getResources().getDrawable(R.drawable.ic_info_outline);
+        aboutDrawable.setColorFilter(
+                ContextCompat.getColor(this, R.color.colorAccent_themeDark),
```

Comment:
That color is different from the other icons in the drawer.

Reply:
Yes, unfortunately. However, I would like to avoid introducing another png file or a separate grey fr the navigation drawer icons. To streamline the color I've proposed PR #434 :) Is that ok for you? :]

Reply:
Yes, thanks.
Do you think we should use the same color for all themes?

Reply:
I think the color is good for all themes. You save the color here https://github.com/openhab/openhab.android/pull/434/files#diff-07e4a6e281af02db331ff3f81d9f86d8R1395 and query it for the about drawable again. Could you rebase #434 on top of this pr?

Reply:
Hmm, rebasing this PR based on 434 would include all changes from the one PR in this PR; which are unrelated :/ I think, that there will be merge conflicts in one PR, dependant on which one is merged first, and I would resolve the conflicts with keeping in mind, that the color is already saved, ok? :)

And to the theme thing: Yes, the color should be ok, I've switched all themes and any theme looks good with the grey color :)

Reply:
Ok.

Reply:
The other pr is merged now.